### PR TITLE
New database system

### PR DESCRIPTION
**Pull request by Monaldcry7788**
I know that the pr is a little confusional, but idk why this appear.

### CODE REVIEW

**Comment by TheUltiOne** [comment]
Why are there 3 different PRs?

File: Scope.Server/Database/Collections/ClientCollection.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Scope.Server.Database.Collections
+{
+    using System;
+    using LiteDB;
+
+    /// <summary>
+    /// A class which contains client information.
```
Comment (NaoUnderscore) [inline Scope.Server/Database/Collections/ClientCollection.cs]: Change to _A tool to handle clients information for database solutions._

File: Scope.Server/Database/Collections/ClientCollection.cs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    /// </summary>
+    internal class ClientCollection
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ClientCollection"/> class.
+        /// </summary>
+        /// <param name="rawId">Client raw id.</param>
```
Comment (NaoUnderscore) [inline Scope.Server/Database/Collections/ClientCollection.cs]: Use 's

File: Scope.Server/Database/Collections/ClientCollection.cs [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            Auth = auth;
+            Username = username;
+            IPAddress = ipAddress;
+            HWID = hwid;
+            FirstJoin = firstJoin;
+            LastJoin = lastJoin;
+        }
+
+        /// <summary>
+        /// Gets the client raw id.
```
Comment (NaoUnderscore) [inline Scope.Server/Database/Collections/ClientCollection.cs]: Use 's

File: Scope.Server/Database/Database.cs [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+        /// <summary>
+        /// Gets or sets clients collection.
+        /// </summary>
+        internal static ILiteCollection<Collections.ClientCollection> ClientCollection { get; set; }
+
+        /// <summary>
+        /// Gets the folder path.
+        /// </summary>
+        internal static string Folder => "Server folder";
```
Comment (NaoUnderscore) [inline Scope.Server/Database/Database.cs]: Server folder? Really e.e?

File: Scope.Server/Database/Database.cs [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+                LiteDatabase = new LiteDatabase(FullPath);
+                ClientCollection = LiteDatabase.GetCollection<Collections.ClientCollection>();
+
+                ClientCollection.EnsureIndex(p => p.RawId, true);
+                ClientCollection.EnsureIndex(p => p.HWID, true);
+
+                Log.Info("Database loaded");
+            } catch (Exception e)
+            {
+                Log.Error($"Error when try to load database:\n {e.StackTrace}\n {e.Message}");
```
Comment (NaoUnderscore) [inline Scope.Server/Database/Database.cs]: Change to _Error whilst trying to load the database..._

File: Scope.Server/Database/Database.cs [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+            }
+        }
+
+        /// <summary>
+        /// Method for close server database.
```
Comment (NaoUnderscore) [inline Scope.Server/Database/Database.cs]: Change to _Closes the database_

File: Scope.Server/Database/Database.cs [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+        /// </summary>
+        internal static string Folder => "Server folder";
+
+        /// <summary>
+        /// Gets the database path.
+        /// </summary>
+        internal static string FullPath => Path.Combine(Folder, "database.db");
+
+        /// <summary>
+        /// Method for load server database.
```
Comment (NaoUnderscore) [inline Scope.Server/Database/Database.cs]: Change to _Opens the database_

File: Scope.sln [hunk 8]
```diff
@@ -11,6 +11,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		.editorconfig = .editorconfig
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Scope.Server", "..\..\source\repos\Scope\Scope.Server\Scope.Server.csproj", "{DC7D4A88-D775-4B7D-A273-578C9B0F476E}"
```
Comment (NaoUnderscore) [inline Scope.sln]: You can't reference your local path

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]

**Review by TheUltiOne — CHANGES_REQUESTED** [review]
Add “The” before “client’s”
Not required but just for it to be better.

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]
You must reference the project in the solution

**Review by NaoUnderscore — APPROVED** [review]

**Review by xRoier — APPROVED** [review]

**Pull request merged**